frontend/src/app/components/block/block.component.ts

```diff
@@ -172,12 +172,13 @@ export class BlockComponent implements OnInit, OnDestroy {
       halvenings--;
     }
   }
-
   pageChange(page: number) {
     const start = (page - 1) * this.itemsPerPage;
     this.isLoadingTransactions = true;
     this.transactions = null;
 
+    document.querySelector('#block-tx-title').scrollIntoView();
```

Comment:
Access the DOM using ViewChild .. the angular way instead of the global querySelector

https://stackoverflow.com/q/38944725

Reply:
Done!